src/main/java/kr/hhplus/be/server/api/payment/PaymentController.java

```diff
@@ -61,8 +61,8 @@ public ResponseEntity<PaymentResponse> payReservation(
             @PathVariable UUID reservationId,
             @RequestHeader(value = "Authorization") String queueToken
     ) throws CustomException {
-        String parsedQueueToken = queueToken.substring("Bearer ".length());
-        paymentInput.payment(PaymentCommand.of(reservationId, parsedQueueToken));
+        queueToken = queueToken.startsWith("Bearer ") ? queueToken.substring("Bearer ".length()) : queueToken;
```

Comment:
```suggestion
        String bearerQueueToken = queueToken.startsWith("Bearer ") ? queueToken.substring("Bearer ".length()) : queueToken;
```

불변성을 지켜주면 더 좋을것 같네요!